### ISSUE TITLE
[Db] Optional modules should be on a separated DBs

### ISSUE THREAD

**Issue by henrique-silva** [lnls-dig/rssmx100a-epics-ioc#4]
Despite having the same model number (e.g. SMA 100 A), some equipments have optional hardware features, such as FM modulation, Pulse Modulation and Clock Synthesis.
When the SMA receives commands for those modules, it gives the SCPI error -113, meaning that the message is syntactically correct, but the command is not present. After that, it looks like the generator enters an error state and stops accepting/replying to all commands.

A good way to address this problem is to separate these optional modules into different DBs, so they  can just be loaded if needed on the st.cmd

**Comment by lerwys** [comment]
Absolutely! This would be a nice addition. If you have something please submit it as a pull request. Cheers!
